backend/app/api/endpoints/admin/system_config.py

```
@@ -213,3 +215,46 @@ async def update_slogan_tips_config(
         slogans=config_data.slogans,
         tips=config_data.tips,
     )
+
+
+# ==================== Admin Setup Wizard Endpoints ====================
+
+
+@router.post("/setup-complete", response_model=AdminSetupCompleteResponse)
+async def mark_admin_setup_complete(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)